### PR TITLE
new tests added for 1.4.4

### PR DESCRIPTION
**Pull request by sridevi-15**
#### Fixes #.

- [x] Ran `flake8`
- [x] Ran `run_repo_tests.sh`

#### Changes proposed in this pull request:

- New tests added for 1.4.4

### CODE REVIEW

File: keywords/exceptions.py [hunk 1]
```diff
@@ -64,3 +64,7 @@ class DockerError(Error):
 
 class FeatureSupportedError(Error):
     pass
+
+
+class ConnectionError(Error):
```
Comment (hrajput89) [inline keywords/exceptions.py]: what is the class for?

File: testsuites/listener/shared/client_sg/test_replication.py [hunk 2]
```diff
@@ -1584,6 +1585,69 @@ def test_replication_with_10_attachments(setup_client_syncgateway_test):
...
+    sg_config = sync_gateway_config_path_for_mode("listener_tests/listener_tests", sg_mode)
+    c = cluster.Cluster(config=cluster_config)
+    c.reset(sg_config_path=sg_config)
+
+    client = MobileRestClient()
+    client.create_database(ls_url, ls_db)
+    client.create_user(sg_url_admin, sg_db, "autotest", password="password", channels=channels)
+    session = client.create_session(sg_url_admin, sg_db, "autotest")
+
+    # add dosc to mobile rest client
```
Comment (hrajput89) [inline testsuites/listener/shared/client_sg/test_replication.py]: typo "docs"

File: testsuites/listener/shared/client_sg/test_replication.py [hunk 3]
```diff
@@ -1584,6 +1585,69 @@ def test_replication_with_10_attachments(setup_client_syncgateway_test):
...
+    assert len(sg_docs) == num_docs
+
+    # Start continuous push replication from ls_db -> sg_db
+    repl_id = client.start_replication(
+        url=ls_url,
+        continuous=True,
+        from_url=sg_url_admin, from_db=sg_db,
+        to_db=ls_db
+    )
+    time.sleep(180)
```
Comment (hrajput89) [inline testsuites/listener/shared/client_sg/test_replication.py]: why this sleep needed if we are anyways waiting for idle status?

File: testsuites/listener/shared/client_sg/test_replication.py [hunk 4]
```diff
@@ -1743,3 +1807,39 @@ def add_2MB_docs(url, db, number, id_prefix, auth=None, channels=None, attachmen
...
+
+    log_info("Running 'test_initial_push_replication'")
+    log_info("ls_url: {}".format(ls_url))
+
+    client = MobileRestClient()
+    client.create_database(url=ls_url, name=ls_db)
+
+    # Create 'num_docs' docs on LiteServ
+    t = time.time()
+    t = t + 19008000
```
Comment (hrajput89) [inline testsuites/listener/shared/client_sg/test_replication.py]: change it to 60 * 60 * 24 * 60 for better readability

File: keywords/exceptions.py (hunk 1 above)
Comment (sridevi-15) [reply keywords/exceptions.py]: In test_doc_expiry_exceeded_200days, I need to catch the connection Error in python test. I need to add this ConnectionError to let Python script to identify as exception

File: testsuites/listener/shared/client_sg/test_replication.py (hunk 2 above)
Comment (sridevi-15) [reply testsuites/listener/shared/client_sg/test_replication.py]: done

File: testsuites/listener/shared/client_sg/test_replication.py (hunk 3 above)
Comment (sridevi-15) [reply testsuites/listener/shared/client_sg/test_replication.py]: do not remember why I have added. As status check is there whether it is idle or not, I removed sleep.

**Pull request merged**